src/components/modals/ant-management/UpgradeAntModal/UpgradeAntModal.tsx

```diff
@@ -151,36 +145,11 @@ function UpgradeAntModal({
           <>
             <div className="flex box-border h-full overflow-hidden w-full flex-col p-4 text-sm text-white">
               <div className="flex scrollbar h-full min-h-[120px] border-b-[1px] border-dark-grey pb-4 mb-4 overflow-y-scroll scrollbar-thumb-primary-thin scrollbar-thumb-rounded-full scrollbar-w-2">
-                <ReactMarkdown
-                  className={'h-full'}
-                  children={changelog}
-                  components={{
-                    h1: ({ children }) => (
-                      <div>
-                        {children == 'Changelog' ? (
-                          <></>
-                        ) : (
-                          <h1 className="pb-4 text-2xl">{children}</h1>
-                        )}
-                      </div>
-                    ),
-                    h2: ({ children }) => (
-                      <h2 className="p-2 text-xl">{children}</h2>
-                    ),
-                    h3: ({ children }) => (
-                      <h3 className="pl-2 text-lg">{children}</h3>
-                    ),
-                    li: ({ children }) => (
-                      <li className="ml-8 list-disc text-sm text-grey">
-                        {children}
-                      </li>
-                    ),
-                  }}
-                />
+                <AntChangelog />
               </div>
 
               <span
-                className="flex flex-row items-center py-4"
+                className="flex flex-row items-center text-[16px] py-4"
```

Comment:
Should be able to use text-base here rather than px

Reply:
base is 14px